AIFG-3D-Connect-4/TurnHandler.h

```diff
@@ -2,6 +2,7 @@
 #define TURN_HANDLER_H
 #include "ConsoleInput.h"
 #include "FourTechAI.h"
+#include "SfmlInput.h"
```

Comment:
This should only include the Input base class, it doesn't need to know the details of any of the implementations